Sources/Sentry/include/SentryPrivate.h

```diff
@@ -1,15 +1,13 @@
 // Sentry internal headers that are needed for swift code
-
 #import "SentryBaggage.h"
-#import "SentryBaseIntegration.h"
-#import "SentryCoreGraphicsHelper.h"
 #import "SentryDispatchQueueWrapper.h"
-#import "SentryFileManager.h"
-#import "SentryGlobalEventProcessor.h"
 #import "SentryNSDataUtils.h"
 #import "SentryRandom.h"
 #import "SentryReplayRecording.h"
 #import "SentryReplayType.h"
 #import "SentrySdkInfo.h"
 #import "SentryStatsdClient.h"
 #import "SentryTime.h"
+
+// Headers that also import SentryDefines should be at the end of this list
```

Comment:
`l`: It would be great to explain why in this comment.

Reply:
I dont know. 
But this is the only way it will compile 🫤